drivers/sensor/ens210/ens210.c

```diff
@@ -107,13 +107,13 @@ static int ens210_channel_get(struct device *dev,
 		temp_frac -= 273150000;
 
 		val->val1 = temp_frac / 1000000;
-		val->val2 = temp_frac - val->val1;
+		val->val2 = temp_frac - val->val1 * 1000000;
```

Comment:
why not doing:

  val->val2 = temp_frac % 1000000;

This is basically what all the drivers are doing.
Does it make sense to you?

Reply:
Sure